### PR TITLE
Edit pass over the current 6.0 NEWS entries. [nomail] [skip ci]

### PR DESCRIPTION
**Pull request by ckreibich**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -67,10 +67,16 @@ Breaking Changes
   If you see opportunities to extend ``repo-info.json`` with further information,
   please get in touch.
 
+- Plugin authors should raise the minimum required CMake version to 3.15 to
+  ensure compatibility with new CMake scaffolding included in this
+  release. Older versions will trigger a warning at configuration time and,
+  depending on the functionality included in the plugin, may trigger subsequent
+  errors.
```
Comment (timwoj) [inline NEWS:74]:
```suggestion
  errors during configuration or the build.
```

Just making this clear that this is a build-time issue, not a run-time issue.

File: NEWS [hunk 2]
```diff
@@ -80,15 +86,16 @@ New Functionality
...
 
-  When a recent version of the libnode package is installed, the externally
-  maintained ZeekJS plugin (https://github.com/corelight/zeekjs) is automatically
-  included as a builtin plugin. This allows Zeek to load and execute execute
-  JavaScript code located in ``.js`` or ``.cjs`` files. When no such files are
-  passed to Zeek, the JavaScript engine and Node.js environment aren't initialized
-  and there is no runtime impact.
+  When building Zeek on a system that features a recent (16.13+) version of the
+  libnode package with development headers, Zeek automatically includes the
+  externally maintained ZeekJS plugin (https://github.com/corelight/zeekjs) as a
```
Comment (timwoj) [inline NEWS:91]:
```suggestion
  externally-maintained ZeekJS plugin (https://github.com/corelight/zeekjs) as a
```

File: NEWS [hunk 3]
```diff
@@ -221,9 +227,10 @@ Changed Functionality
 - Zeek's CMake scaffolding has received an overhaul for modernizing the build
   system and to make it easier to maintain going forward. Plugins can now use a
   declarative interface for adding all sources, BIFs, etc. in one block instead
-  of using the previous begin/end functions. However, the old functions still
-  exist for backwards compatibility. Since Zeek uses newer CMake features now,
-  plugin authors should also raise their minimum required CMake version to 3.15.
+  of using the previous begin/end functions. While the old plugin functions
+  still exist for backward compatibility, the underlying codebase requires newer
+  CMake features. Plugin authors should raise their minimum required CMake
```
Comment (timwoj) [inline NEWS:232]: Do we need to mention this version requirement again after we already mentioned it in an earlier section?

**Review by timwoj — CHANGES_REQUESTED** [review]

File: NEWS (hunk 3 above)
Comment (ckreibich) [reply NEWS:232]: I figured it doesn't hurt — we've previously erred on the side of not mentioning such things enough.

File: NEWS (hunk 1 above)
Comment (ckreibich) [reply NEWS:74]: Good one, yeah.

File: NEWS (hunk 2 above)
Comment (ckreibich) [reply NEWS:91]: You wouldn't believe the number of beers academics have consumed debating that one. ;-)

**Review by timwoj — APPROVED** [review]

**Pull request merged**